3_javascript/javascript1/week1/A future age calculator.js

```diff
@@ -0,0 +1,4 @@
+const yearOfBirth= 1996;
+const yearFuture= 2050;
+const age= yearFuture - yearOfBirth;
+console.log("You will be " + age +"years old in " + yearFuture);
```

Comment:
👍 Great use of const for values that don't change!
💡 Consider using template literals to make the string concatenation more readable